--- a/OpenRGB.pro	2022-05-02 15:26:15.375947000 +0200
+++ b/OpenRGB.pro	2022-05-02 15:29:01.997135000 +0200
@@ -449,8 +449,6 @@
 # Linux-specific Configuration                                                                  #
 #-----------------------------------------------------------------------------------------------#
 contains(QMAKE_PLATFORM, linux) {
-    TARGET = $$lower($$TARGET)
-
     HEADERS -= $$CONTROLLER_H_WIN
 
     HEADERS +=                                                                                  \
--- a/qt/org.openrgb.OpenRGB.desktop
+++ b/qt/org.openrgb.OpenRGB.desktop
@@ -4,7 +4,7 @@ Version=1.5
 Name=OpenRGB
 Comment=Control RGB lighting
 Icon=org.openrgb.OpenRGB
-TryExec=openrgb
-Exec=openrgb
+TryExec=OpenRGB
+Exec=OpenRGB
 Terminal=false
 Categories=Utility;
